Frameworks/Foundation/NSMutableURLRequest.mm

```diff
@@ -32,30 +32,18 @@ - (void)_lazyInitAllHTTPHeaderFields {
  @Status Interoperable
 */
 - (void)addValue:(NSString*)value forHTTPHeaderField:(NSString*)headerName {
```

Comment:
Review updated

---
In reply to: [73719210](https://github.com/Microsoft/WinObjC/pull/825#discussion_r73719210) [](ancestors = 73719210)